website/redirects.js

```diff
@@ -5212,4 +5172,468 @@ module.exports = [
+    destination: "/consul/docs/architecture/security",
+    permanent: true,
+  },
+  {
+    source: "/consul/docs/connect/config-entries/partition-exports",
+    destination: "/consul/docs/reference/config-entry/exported-services",
+    permanent: true,
+  },
+  {
+    source: "/consul/docs/connect/configuration",
+    destination: "/consul/docs/fundamentals/config-entry",
+    permanent: true,
+  },
+  {
+    source: "/consul/docs/connect/gateways/api-gateway/configuration/:slug*",
+    destination: "/consul/docs/north-south/api-gateway",
+    permanent: true,
+  },
+  {
+    source: "/consul/docs/connect/gateways/ingress-gateway/:slug*",
+    destination: "/consul/docs/north-south/ingress-gateway",
+    permanent: true,
+  },
+  {
+    source: "/docs/k8s/crds",
+    destination: "/consul/docs/fundamentals/config-entry",
+    permanent: true,
+  },
+  {
+    source: "/consul/docs/:version(v1\\.(?:18|19|20)\\.x)/agent/telemetry",
+    destination: "/consul/docs/:version/agent/monitor/telemetry",
+    permanent: true,
+  },
+  {
+    source: "/docs/connect/intentions",
+    destination: "/consul/docs/secure-mesh/intention",
+    permanent: true,
+  },
+  {
+    source: "/consul/docs/security/acl/acl-legacy",
+    destination: "/consul/docs/secure/acl/legacy",
+    permanent: true,
+  },
+  {
+    source: "/consul/docs/:version(v1\\.(?:18|19|20)\\.x)/connect/transparent-proxy",
+    destination: "/consul/docs/:version/k8s/connect/transparent-proxy",
+    permanent: true,
+  },
+  {
+    source: "/consul/docs/:version(v1\\.(?:9|10)\\.x)/agent/services",
+    destination: "/consul/docs/:version/discovery/services",
+    permanent: true,
+  },
+  {
+    source: "/docs/connect/config-entries/service-router",
+    destination: "/consul/docs/reference/config-entry/service-router",
+    permanent: true,
+  },
+  {
+    source: "/consul/docs/guides/sentinel",
+    destination: "/sentinel/docs",
+    permanent: true,
+  },
+  {
+    source: "/consul/docs/v1.20.x/k8s/dns/:slug*",
+    destination: "/consul/docs/manage/dns/forwarding/k8s",
+    permanent: true,
+  },
+  {
+    source: "/docs/enterprise/admin-partitions",
+    destination: "/consul/docs/multi-tenant/admin-partition",
+    permanent: true,
+  },
+  {
+    source: "/consul/docs/:version(v1\\.(?:18|19|20)\\.x)/nia/network-drivers/terraform-cloud",
+    destination: "/consul/docs/:version/nia/network-drivers/hcp-terraform",
+    permanent: true,
+  },
+  {
+    source: "/consul/docs/security/acl/acl-tokens",
+    destination: "/consul/docs/secure/acl/token",
+    permanent: true,
+  },
+  {
+    source: "/consul/docs/security/acl/acl-system",
+    destination: "/consul/docs/secure/acl/reset",
+    permanent: true,
+  },
+  {
+    source: "/consul/docs/security/acl/migrate-tokens",
+    destination: "/consul/docs/secure/acl/legacy",
+    permanent: true,
+  },
+  {
+    source: "/docs/enterprise",
+    destination: "/consul/docs/enterprise",
+    permanent: true,
+  },  
```

Comment:
Per the build logs, this redirect is invalid due to the source not being prefixed with `/consul`. Given that doing so would result in a page redirecting to itself, it would probably be best to just remove the redirect altogether.

```suggestion
```